test/xpu_api/utilities/pairs/pairs.pair/noexcept_swap_g.pass.cpp

```diff
@@ -0,0 +1,66 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+
+#include "support/test_config.h"
+
+#include <oneapi/dpl/utility>
+
+#include "support/utils.h"
+
+// #include "testsuite_struct.h"        // KSATODO required to remove?
```

Comment:
@rarutyun @ValentinaKats May we use this classes?
```C++
                typedef dpl::pair<short, NoexceptMoveAssignClass> tt3;
                typedef dpl::pair<short, NoexceptMoveConsClass> tt4;
                typedef dpl::pair<NoexceptMoveConsClass, NoexceptMoveConsClass> tt5;
                typedef dpl::pair<NoexceptMoveConsNoexceptMoveAssignClass, 
NoexceptMoveConsNoexceptMoveAssignClass> tt6;
                typedef dpl::pair<NoexceptMoveConsNoexceptMoveAssignClass, float> tt7;
                typedef dpl::pair<NoexceptMoveConsNoexceptMoveAssignClass, NoexceptMoveConsNoexceptMoveAssignClass> tt8;
```
It's from test suite - see my comments in code below.

Reply:
It's funny, but all tests from this PR have compiled without errors with this stub code:
```C++
struct TestStruct
{
};
```